### ISSUE TITLE
[Skype for Business] duplicate message appeared in SFB mobile app

### ISSUE THREAD

**Issue by jessedemobot** [Microsoft/BotBuilder#4122]
## Bot Info
* SDK Platform:  .NET
* SDK Version: 3.12.2.4
* Active Channels: Skype for Business
* Deployment Environment: Azure Bot Service

## Issue Description
Outbound message in Skype for Business Mobile App appeared twice in 5-10 minutes.

## Code Example
string trustServiceUri = "https://api.skypeforbusiness.com/platformservice/botframework";
MicrosoftAppCredentials.TrustServiceUrl(trustServiceUri);
ConnectorClient connector = new ConnectorClient(new Uri(trustServiceUri));
List< ChannelAccount > participants = new List< ChannelAccount >();
participants.Add(new ChannelAccount("sip:usera@contoso.com", "usera"));
ConversationParameters parameters = new ConversationParameters(true, new ChannelAccount( "sip:bot@contoso.com", "LEO"), participants, "");
ConversationResourceResponse response = connector.Conversations.CreateConversationAsync(parameters).Result;

ConnectorClient connectorSend = new ConnectorClient(new Uri(response.ServiceUrl));
IMessageActivity msg= Activity.CreateMessageActivity();
msg.Recipient = new ChannelAccount("sip:usera@contoso.com", "usera");
msg.Text = "reminder message14";
msg .Locale = "en-Us";
msg.From = new ChannelAccount("sip:bot@contoso.com", "LEO");
msg.Type = ActivityTypes.Message;
msg.Timestamp = DateTime.UtcNow;
msg.ChannelId = "skypeforbusiness";
msg.ServiceUrl = response.ServiceUrl;
msg.Conversation = new ConversationAccount(isGroup: true, id: response.Id, name: null);
var result = connectorSend.Conversations.SendToConversationAsync((Activity)msg, response.Id).Result;

## Reproduction Steps
1. Sign-in Skype for Business mobile app;
2. Run the example code;
3. Wait 8-10 minutes

## Expected Behavior
The outbound message should be appeared only once in the Mobile app.

## Actual Results
![sfb](https://user-images.githubusercontent.com/35955037/35600825-eabca170-066a-11e8-8e39-4923eca8bd1d.jpg)

**Comment by cleemullins** [comment]
Thank you for opening an issue against the Bot Framework SDK v3. As part of the Bot Framework v4 release, we’ve moved all v3 work to a new repo located at https://github.com/microsoft/botbuilder-v3. We will continue to support and offer maintenance updates to v3 via this new repo.
 
From now on, https://github.com/microsoft/botbuilder repo will be used as hub, with pointers to all the different SDK languages, tools and samples repos.
 
As part of this restructuring, we are closing all tickets in this repo.
 
For defects or feature requests, please create a new issue in the new Bot Framework v3 repo found here:
https://github.com/microsoft/botbuilder-v3/issues


For Azure Bot Service Channel specific defects or feature requests (e.g. Facebook, Twilio, Teams, Slack, etc.), please create a new issue in the new Bot Framework Channel repo found here:
https://github.com/microsoft/botbuilder-channels/issues
 
For product behavior, how-to, or general understanding questions, please use Stackoverflow.
https://stackoverflow.com/search?q=bot+framework


Thank you.
 
The Bot Framework Team
